js/app.js

```diff
@@ -2,299 +2,100 @@
+var storeList = [];
+var cookiesTable = document.getElementById('cookies');
 
-var firstAndPike = {
-	// min and max customers per hour
-	minCustPerHour: 23,
-	maxCustPerHour: 65,	
-	avgCookPerCust: 6.3,
-  
-  // methodToGenerateRandomCustomers		
-	calcCustPerHour: function() {
-		//Create random number generator function, call here to save code in all objects
-		for (var i = 0; i < times.length; i++) {
-			this.custPerHour.push(Math.floor(Math.random() * (this.maxCustPerHour - this.minCustPerHour)) + this.minCustPerHour);
-		}
-	},
-	// Array customers each Hour
-	custPerHour: [],
-	
-	// Calculate cookies sold each hour and Total for Day
-  calcCookies: function() {
-		for (var i = 0; i < this.custPerHour.length; i++) {
-			var hourlyCookies = (this.custPerHour[i] * this.avgCookPerCust);
-			this.cookiesPerHour.push(hourlyCookies);
-			this.dayTotalCookies += hourlyCookies;
-    }
-  },
-  // Array cookies sold per hour and Total for day
-  cookiesPerHour: [],
-  dayTotalCookies: 0,
-  
-	// renderMethod
-	render: function() {
-		this.calcCustPerHour();
-		this.calcCookies();
-		var ulEl = document.getElementById('firstAndPike');		
-		for (var i = 0; i < this.cookiesPerHour.length; i++) {
-			// create an element
-			var liEl = document.createElement('li');
-			// give it content
-			liEl.textContent = times[i] + ': ' + Math.round(this.cookiesPerHour[i]);
-			// append it to the DOM
-			// parent.appendChild(child);
-			ulEl.appendChild(liEl);
-		}
-		// create an element
-		var liEl = document.createElement('li');
-		// render dayTotalCookies
-		liEl.className = ('total');
-		liEl.textContent = 'Total: ' + Math.round(this.dayTotalCookies) + ' cookies';
-		// Append to DOM
-		ulEl.appendChild(liEl);
-	}
+// object constructor for each store
+function Store(storeName, minCustPerHour, maxCustPerHour, avgCookPerCust) {
+  this.storeName = storeName;
+  this.minCustPerHour = minCustPerHour;
+  this.maxCustPerHour = maxCustPerHour;
+  this.avgCookPerCust = avgCookPerCust;
+  this.custPerHour = [];
+  this.cookiesPerHour = [];
+  this.dayTotalCookies = 0;
+  storeList.push(this);
+  this.calcCustPerHour();
+  this.calcCookies(); //
+}
+
+// Methods within the constructor
+Store.prototype.calcCustPerHour = function() {
+  //Create random number generator function, call here to save code in all objects
+  for (var i = 0; i < times.length; i++) {
+    this.custPerHour.push(Math.floor(Math.random() * (this.maxCustPerHour - this.minCustPerHour)) + this.minCustPerHour);
+  }
+};
+Store.prototype.calcCookies = function() { // Calculates cookies sold each day
+  for (var i = 0; i < this.custPerHour.length; i++) {
+    var hourlyCookies = Math.ceil(this.custPerHour[i] * this.avgCookPerCust);
+    this.cookiesPerHour.push(hourlyCookies);
+    this.dayTotalCookies += hourlyCookies;
+  }
 };
 
+// Table Element Render Functions
+Store.prototype.render = function() {
+  this.calcCookies();
```

Comment:
You should probably have `this.calcCookies()` either in the constructor _or_ in the this `render` method, not both.